City-of-Helsinki/berth-reservations-admin
# VEN-378 | Fix silent token renew

## Description :sparkles:
- Add and `html` document to be loaded in iframe to process the token silent-renew
- Fix and write unit tests

## Issues :bug:

### Closes :no_good_woman:
[VEN-378](https://helsinkisolutionoffice.atlassian.net/browse/VEN-378): Support silent token renew

### Related :handshake:

## Testing :alembic:

### Automated tests :gear:️
https://github.com/City-of-Helsinki/berth-reservations-admin/blob/0d14d11cc8f1b6b055ebfe4b552efb69a315a1f6/src/domain/auth/authService.test.js#L7

### Manual testing :construction_worker_man:
- Log in to the service
- Wait for more than an hour 🕐 
- You should remain logged in and the session should be renewed silently 

## Screenshots :camera_flash:

## Additional notes :spiral_notepad:
